fhir-persistence-cos/src/main/java/com/ibm/fhir/persistence/cos/client/COSConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.cos.client;
+
+
+/**
+ * Constants related to our COS connection
+ */
+public class COSConstants {
+    // the IBM COS API key or S3 access key.
+    public static final String COS_API_KEY = "cos.api.key";
+
+    // the IBM COS service instance id or S3 secret key.
+    public static final String COS_SRVINSTID = "cos.srvinstid";
+
+    // the IBM COS or S3 End point URL.
+    public static final String COS_ENDPOINT_URL = "cos.endpoint.url";
+
+    // the IBM COS or S3 location.
+    public static final String COS_LOCATION = "cos.location";
+
+    // the IBM COS or S3 bucket name to import from.
+    public static final String COS_BUCKET_NAME = "cos.bucket.name";
+
+    // if use IBM credential(Y/N), default(Y).
+    public static final String COS_CREDENTIAL_IBM = "cos.credential.ibm";
+
+    public static final String COS_REQUEST_TIMEOUT = "cos.request.timeout";
+    
+    public static final String COS_SOCKET_TIMEOUT = "cos.socket.timeout";
+    
+    // The max keys to return per list objects request
+    public static final String COS_MAX_KEYS = "cos.max.keys";
+
```

Comment:
Please use the same form as 


fhirServer/bulkdata/core/cos/requestTimeout | number | The request timeout in second for the COS client
-- | -- | --
fhirServer/bulkdata/core/cos/socketTimeout | number | The socket timeout in second for the COS client



All of these have been deprecated and replaced in bulkdata, and I'd like to not propagate them.  https://github.com/IBM/FHIR/blob/main/docs/src/pages/guides/FHIRServerUsersGuide.md#511-property-descriptions